app/views/decidim/proposals/admin/proposals/_proposal-tr.html.erb

```diff
@@ -105,6 +105,29 @@
 
     <%= resource_permissions_link(proposal) %>
 
-   <%= icon_link_to "eye", resource_locator(proposal).path, t("actions.preview", scope: "decidim.proposals.admin"), class: "action-icon--preview", target: :blank %>
+    <%= icon_link_to "eye", resource_locator(proposal).path, t("actions.preview", scope: "decidim.proposals.admin"), class: "action-icon--preview", target: :blank %>
+
+    <% if Decidim::ReportingProposals.allow_admins_to_hide_proposals && !proposal.reported? %>
+      <%= cell("decidim/reporting_proposals/admin/hide_flag_modal", proposal, modal_id: "flagModal#{proposal.id}") %>
+      <button type="button" class="link-alt"
+              data-open="<%= current_user.present? ? "flagModal#{proposal.id}" : "loginModal" %>"
+              title="<%= t("report", scope: "decidim.proposals.proposals.show") %>"
+              aria-controls="<%= current_user.present? ? "flagModal#{proposal.id}" : "loginModal" %>"
+              aria-haspopup="true"
+              tabindex="0"
+              style="color:#97a2b2; cursor: pointer">
+        <%= icon "flag", aria_hidden: true, class: "action-icon--preview", role: "img", "aria-hidden": true %>
+        <span class="show-for-sr">
+      <%= t("report", scope: "decidim.proposals.proposals.show") %>
+    </span>
+      </button>
+    <% end %>
+    <% if proposal.reported? %>
+      <%= icon_link_to "eye", hide_proposal_path(id: proposal),
```

Comment:
```suggestion
      <%= icon_link_to "trash", hide_proposal_path(id: proposal),
```
I think we can be more direct and directly show the "trash" icon (no need to cross it with a line).